src/App.tsx

```diff
@@ -1,10 +1,53 @@
 import "./App.scss";
+import Box from "@mui/material/Box";
+import { Logo } from "./components/logo/Logo";
+import sunIcon from "./assets/icon-sun.svg";
+import { Avatar } from "./components/avatar/Avatar";
 
 function App() {
   return (
-    <>
-      <div className="app"></div>
-    </>
+    <div className="app">
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          position: "relative",
+          bgcolor: "#1f213a",
```

Comment:
casing is wrong and it should be placed in the theme object - we did this in one of the previous projects